platform/src/hyper/mod.rs

```diff
@@ -91,8 +92,10 @@ pub async fn request<T: Serialize, U: DeserializeOwned>(
         .method(method)
         .uri(uri)
         .header(CONTENT_TYPE, content_type.to_string())
+        .header(USER_AGENT, String::from("SBOM Harbor CLI"))
```

Comment:
So, since this is a general purpose library, it could be used by things other than the CLI right?  If we need/want a User-Agent header I think we need to add it as a parameter to the functions.

Reply:
This came from @qtpeters github branch, he may already have addressed this there